src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import './App.css'
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import HomePage from './components/homePage/HomePage';
+import MyAppointments from './components/appointmentInfo/MyAppointments';
+import DoctorList from './components/doctorInfo/DoctorList';
+import DeleteDoctor from './components/doctorInfo/DeleteDoctor';
+import PatientList from './components/forSuperUser/PatientList';
+import Appointment from './components/appointmentInfo/Appointment';
+import AppointmentList from './components/forSuperUser/AppointmentList';
+import UserList from './components/forSuperUser/UserList';
+import DoctorDetails from './components/doctorInfo/DoctorDetails.jsx';
+import ModalAppoint from './components/appointmentInfo/ModalAppoint';
+
+function App() {
```

Comment:
- [ ] The project is not working on my end because it won't let me create a new user:

![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/83ce66e4-8be7-4500-a3e6-23eea94069ea)

Steps: Register a new user (multiple roles tried), and then try to log in.

I tested it on the live demo, and locally on both Firefox and Chrome.

If this is a bug, kindly fix it so your users can get into your app.  

If this is NOT a bug, this means your project requires special procedures to be run correctly (like a .env file).  If this is true, kindly add the instructions to your readme about how to correctly run your frontend app.

That way, your users will be able to run it smoothly. 👍 

Tip: If this works on your end, this could be that your local storage is already populated, but this is not true for new users.  Therefore, to correctly debug this, try to run your project and create a user on an incognito tab. 

_note: This means I wasn't able to get into the app to review the functionality.  This will be taken a look at on the next review._

Reply:
We have checked front-end deployment on render. We are able to signup new user. The Signup user can be seen in the back-end API on the render. Newly signup user can login successfully into the application as well as can be seen in the screen shot below.

![signup](https://github.com/baqar-abbas/appointment-app-front-end/assets/107859883/4d4edd54-b55d-4ab4-8f80-4ff2934fafd8)
![userCreated](https://github.com/baqar-abbas/appointment-app-front-end/assets/107859883/162322cb-1bb2-4424-bdbd-e3bc0c0e4828)
![signinUser](https://github.com/baqar-abbas/appointment-app-front-end/assets/107859883/e08ce3a6-2922-4f1f-9464-9a4d91235cb1)
![newUserSignedIn](https://github.com/baqar-abbas/appointment-app-front-end/assets/107859883/9630070a-e834-433a-9a47-f24018906692)

Please note: It might be a delay from Render since we are on a free deployment, during peak hours from my experience,  you might face unexpected behaviours from Render, but refresh the page and sign in. The App does not have a bug I can even create users locally by running the front end app on my machine.

Reply:
I have also tried signup using local host rails backend api links. I can signup and create users and then sign in using the newly created user. Screen shot attached below for your kind reference.

![userCreation](https://github.com/baqar-abbas/appointment-app-front-end/assets/117091923/aee157b9-9471-44a4-8203-f00a5056c686)
![userCreatedinDb](https://github.com/baqar-abbas/appointment-app-front-end/assets/117091923/bbf51abc-88ba-4062-84a7-f11f9dd6d0c1)
![loggingIn](https://github.com/baqar-abbas/appointment-app-front-end/assets/117091923/53569be6-5b1f-42b3-bcd7-f2c7a7a137c1)
![userLoggedIn](https://github.com/baqar-abbas/appointment-app-front-end/assets/117091923/aba6e942-7fb1-4514-8d0f-06003770c516)

Reply:
I have also added paths in comments in slice files and where we are using data fetching from apis. You can also switch between local host paths and rendered deployed backend paths to check the working. Screen shot attached below for your kind reference.
![apiPaths](https://github.com/baqar-abbas/appointment-app-front-end/assets/117091923/08c257c8-404f-4691-a522-83db182e5903)

Reply:
You can also use the below created users for logging in and checking the functionality of the app.

name: shihab
email: shihab@appoint.com
password: 123456
role; patient

name: baqar
email baqar@appoint.com
password: 123456
role: admin

Reply:
Hey @baqar-abbas, @Amazinggracee:  I've been monitoring the repo activity because I was really curious about what was the problem.

This change did the trick:

![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/b961dc04-3113-4ec7-a55e-8d55b3a375ac)

I am now able to create users (locally) using the frontend:


![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/dfd7fb1c-2644-4970-b986-626fbc428e70)


![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/094548c6-0c7a-49dd-a269-7ba3c2afad6d)

![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/63cf8a2e-241a-4153-af4f-2c8244f59876)


![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/7d363b3a-6dd9-4d06-8510-182f69caf5a4)